en/docs/ai-gateway/multi-model-routing/load-balancing.md

```diff
@@ -23,6 +23,11 @@ You can enforce round robin based load balancing for your AI API by attaching th
 
     [![Model Round Robin Policy]({{base_path}}/assets/img/learn/ai-gateway/attach-model-round-robin-policy.png){: style="width:90%"}]({{base_path}}/assets/img/learn/ai-gateway/attach-model-round-robin-policy.png)
 
+!!! note "AWS Bedrock Configuration"
+    When configuring **AWS Bedrock** as your provider for round robin routing, you must select both the **Provider** (model family) and the **Model** for each model entry. The **Provider** dropdown will show the model families you have configured in the Admin Portal (such as Meta, Anthropic, DeepSeek, etc.). After selecting a provider, the **Model** dropdown will list the specific models available under that provider.
```

Comment:
We can make this message a bit generic to say, when providing a multi model provider service, we can do this without giving one specifically for Bedrock. There can be other ones like Azure model foundry